traitsui/wx/ui_panel.py

```diff
@@ -1011,7 +1011,7 @@ def add_items(self, content, panel, sizer):
             item_sizer.Add(
                 control,
                 growable,
-                flags | layout_style | wx.ALIGN_CENTER_VERTICAL,
```

Comment:
I think this is now fixed on master. Could you merge master to this branch please?